alf/examples/grid_search.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
```

Comment:
we can move this and main.py to another directory since they does not look like examples. 'bin' might be a good choice.

Reply:
yeah, i'll do this in a seperate change